libraries/AP_Common/missing/fenv.h

```diff
@@ -46,3 +46,29 @@ inline int fedisableexcept(unsigned int excepts)
 }
 
 #endif
+
+#if (defined(HAVE_FEENABLEEXCEPT) && !HAVE_FEENABLEEXCEPT) && !defined(__APPLE__)
```

Comment:
won't this break the build on macos as functions won't be defined?

Reply:
Unless my boolean logic, this won't be define on macos. 
But those functions get their own implementation on macos upper on this file